app/_data/background.yaml

```diff
@@ -1,18 +1,33 @@
 - section:
-    title: What is Caselaw Access Project?
+    title: Why do we need the Caselaw Access Project?
+    text: |
+      Our common law - the written decisions issued by our state and federal courts - is not freely accessible online. This lack of access harms justice and equality and stifles innovation in legal services. 
 
+      Every court decision issued in the U.S. should be available for anyone to read online, and those decisions should be available as datasets that researchers can explore and programmers can make more useful.  
+    hero_image: perma-box-hero-wide.png
+    links: [
+        "[link1](//imgur.com/ipa45yK)", 
+        ]
+- section:
+    title: What is the Caselaw Access Project?
     text: |
-      Working with newly rediscovered African drumming laws to understand how and why the colonial government controlled and criminalized drumming in native African communities
+      CAP is a mass digitization project in which millions of pages of printed court decisions must be scanned and then converted into machine-readable data.
+
+      CAP also is an open data project in which millions of text files, and lots of data about those text files, must be made accessible online to people who want to read them or interact with them using computer programs.
     hero_image: perma-box-hero-wide.png
     links: [
-        "[link1.org](http://www.africandrumminglaws.org/)",
-        "[link2.org](http://www.africandrumminglaws.org/)",
-        "[link3.org](http://www.africandrumminglaws.org/)"
+        "[link1](//imgur.com/ipa45yK)", 
```

Comment:
Are you intending this to be a link outside of the site? Or would you like this image to be embedded?